docs/deploying-airbyte/on-kubernetes.md

```diff
@@ -145,6 +145,14 @@ Now visit [http://localhost:8000](http://localhost:8000) in your browser and sta
 * Volume sizes
   * You can modify `kube/resources/volume-*` files to specify different volume sizes for the persistent volumes backing Airbyte.
 
+### Increasing job parallelism
+
+The number of simultaneous jobs (getting specs, checking connections, discovering schemas, and performing syncs) is limited by a few factors. First of all, the `SUBMITTER_NUM_THREADS` (set in the `.env` file for your Kustimization overlay) provides a global limit on the number of simultaneous jobs that can run across all worker pods. 
```

Comment:
Nitpick. Should `SUBMITTER_NUM_THREADS` be renamed to something more intuitive? Like `GLOBAL_MAX_WORKERS`? If this is too much work, maybe at least add some comments on this variable in `.env`.

Reply:
Added comments. I don't think we have a good story about upgrading critical configuration variables in the OSS project yet, so I'll leave the name as-is for now.